src/Microsoft.ManagedZlib/benchmarks/CompressFiles.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Text;
+
+namespace Microsoft.ManagedZLib.Benchmarks
+{
+    class CompressFiles
+    {
+        
+        private string zipName = @".\resultZip.zip";
+        public void MakeFile(string filename, string message)
+        {
+            //Crea o appendea al final del archivo
```

Comment:
Please translate to english the comments you wish to keep. Otherwise, please remove them.